spark/src/main/scala/geotrellis/spark/io/slippy/FileSlippyTileReader.scala

```diff
@@ -16,16 +16,27 @@
 
 package geotrellis.spark.io.slippy
 
+import geotrellis.vector._
+import geotrellis.raster._
```

Comment:
hm why so many unused imports appeared in all slippy files? O:

Reply:
I'm not sure how those got there....